install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -74,6 +74,9 @@ spec:
 {{- end }}
           - --keepaliveMaxServerConnectionAge
           - "{{ .Values.keepaliveMaxServerConnectionAge }}"
+{{- if .Values.global.proxyProtocol }}
```

Comment:
Is this really a global, mesh-wide flag that applies to everything ? Generally we use metadata or less global settings.

Reply:
will use the environment appoach first, and remove this

Reply:
ok, updated to use an environment var